library/src/test/java/com/datatorrent/lib/db/jdbc/JdbcOperatorTest.java

```diff
@@ -375,14 +411,37 @@ public void testJdbcPojoInputOperator()
     OperatorContextTestHelper.TestIdOperatorContext context = new OperatorContextTestHelper.TestIdOperatorContext(
         OPERATOR_ID, attributeMap);
 
-    insertEventsInTable(10);
```

Comment:
At present, there is just one method that inserts records. Hence did not make a separate function.